Activities/Database/UiPath.Database/DatabaseConnection.cs

```diff
@@ -28,27 +28,34 @@ public class DatabaseConnection : IDisposable
         private const string OracleOdbcDriverPattern = "SQORA";
         private const string OraclePattern = "oracle";
         private const string OracleProvider = "oracle.manageddataaccess.client";
+        bool isWindows = true;
```

Comment:
`private bool _isWindows`

Reply:
Fixed

Reply:
It's not, please add `private` too for consistency.